index.html

```diff
@@ -38,6 +39,43 @@ <h2 class="inspiration_container_title"> 50+ Beautiful rooms <br> inspiration </
             </div>
             </div>
         </section>
+        <section class="footer">

```

Comment:
```suggestion
        <footer class="footer">

```
Лен, это по поводу семантики, класс можно оставить чтобы не переверстывать css. Это на будущее, сейчас я апрувну или заапрувлю, ну короче ты поняла)